### ISSUE TITLE
verilator version 

### ISSUE THREAD

**Issue by xun23333** [ultraembedded/riscv#9]
Your project has strict requirements on the version of Verilator. Due to the changes in translation rules of the new version of Verilator, it cannot be directly applied to your project.  Even if I specifically changed “tb” to make “make” pass, I still had problems with “make run”.  However, when I switched to an older version of Verilator 4.028, everything was fixed.

**Comment by exaithrg** [comment]
Yes, I also found that Verilator 4.210 and 5.008 can't work correctly on this repo. However, using Verilator 4.038, which can be installed directly through "apt install verilator" works. I hope the authors can add support for newer versions of Verilator in the future, thanks

**Comment by ScribblerCoder** [comment]
I agree I just tested it out using Verilator 5.024 (as of this comment's date) installed from source.
